### ISSUE TITLE
The model does not work for images outside the dataset

### ISSUE THREAD

**Issue by BIn-999** [thqiu0419/MLWNet#8]
The images obtained from the training set and the validation set can achieve good deblurring effect, but other images will be output as mosaics. Model robustness issue?

![000000](https://github.com/user-attachments/assets/504dcb70-cae4-45d3-ab33-4978980eb519)
![000001](https://github.com/user-attachments/assets/338337c4-50ae-49fc-8dde-0ce21d1630e6)

**Comment by pyh3552** [comment]
I meet the same problem.
